cpp/tests/test_ide_secir_ageres.cpp

```diff
@@ -0,0 +1,157 @@
+#include "load_test_data.h"
+#include "ide_secir/infection_state.h"
+#include "ide_secir/model.h"
+#include "ide_secir/parameters.h"
+#include "ide_secir/simulation.h"
+#include "memilio/epidemiology/age_group.h"
+#include "memilio/epidemiology/state_age_function.h"
+#include "memilio/math/eigen.h"
+#include "memilio/utils/time_series.h"
+#include "memilio/utils/logging.h"
+#include "memilio/config.h"
+#include "memilio/epidemiology/uncertain_matrix.h"
+#include <gtest/gtest.h>
+#include <vector>
+
+TEST(TestIDEAgeres, compareWithPreviousRun)
```

Comment:
```suggestion
TEST(TestIdeAgeres, compareWithPreviousRun)
```
Just for consistency with other tests.